src/blant.c

```diff
@@ -753,16 +753,17 @@ const char const * const USAGE =
 "	-p windowRepSamplingMethod: (deprecated) one of the below, possibly with prefix [u|U] (meaning unambiguous)\n"\
 "	    MIN (Minimizer); MAX (Maximizer); DMIN (Minimizer With Distance); DMAX (Maximizer with Distance);\n"\
 "	    LFMIN (Least Frequent Minimizer); LFMAX (Least Frequent Maximizer)\n"\
-"	-P windowRepIterationMethods is one of: COMB (Combination) or DFS\n"\
+"	-P windowRepIterationMethods is one of: COMB (Combination) or DFS\n" \
 "	-l windowRepLimitMethod is one of: [suffix N: limit to Top N satisfied graphlets]\n"\
-"	    DEG (graphlet Total Degree); EDGE (1-step away numEdges)";
+"	    DEG (graphlet Total Degree); EDGE (1-step away numEdges)\n"\
+"	-M = multiplicity = max allowed number of ambiguous permutations in found graphlets (M=1 is a special case and means no max)";
```

Comment:
Isn't is M=0 is the special case, and M=1 means "unambiguous"?

Reply:
oh yeah whoops I deleted the wrong one